### PR TITLE
Feat/php hooks otel dd trace

### PR DESCRIPTION
**Pull request by tcarrio**
## This PR

- adds OTel hook
- adds DDTrace hook (and Datadog logo)
- fixes links for existing providers (repo structure changed)

### How to test

View the live page

### CODE REVIEW

File: static/img/datadog-no-fill.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg version="1.2" baseProfile="tiny-ps" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1412 1601" width="1412" height="1601">
+	<title>dd_logo_v_white-svg</title>
+	<style>
+		tspan { white-space:pre }
+		.shp0 { fill: none } 
+		.shp1 { fill: #ffffff }
```
Comment (tcarrio) [inline static/img/datadog-no-fill.svg:6]:
```suggestion
```

Comment (tcarrio) [reply static/img/datadog-no-fill.svg:6]: This probably isn't wanted since it overrides styling in the page theme.

**Comment by tcarrio** [comment]
Those changes worked as expected. See the following screenshot for example:
image

**Comment by tcarrio** [comment]
Added slightly more descriptive descriptions, following suit with JS OTel as a reference.

**Comment by tcarrio** [comment]
Latest changes:
image

**Review by beeme1mr — APPROVED** [review]
Thanks @tcarrio!

**Pull request merged**